src/component/panels/MoleculesPanel/MoleculePanelHeader.tsx

```diff
@@ -290,14 +292,6 @@ export default function MoleculePanelHeader({
       </Toolbar>
 
       <div style={{ flex: 1 }}>{children}</div>
-      {molecules && molecules.length > 0 && (
-        <p style={styles.counter}>
```

Comment:
You are not using this style anymore but I don't think you have removed it from `styles`.